service/src/main/kotlin/fi/espoo/evaka/pedagogicaldocument/PedagogicalDocumentController.kt

```diff
@@ -0,0 +1,259 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.pedagogicaldocument
+
+import fi.espoo.evaka.Audit
+import fi.espoo.evaka.shared.AttachmentId
+import fi.espoo.evaka.shared.ChildId
+import fi.espoo.evaka.shared.PedagogicalDocumentId
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.db.mapColumn
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+import fi.espoo.evaka.shared.security.AccessControl
+import fi.espoo.evaka.shared.security.Action
+import org.jdbi.v3.core.kotlin.mapTo
+import org.jdbi.v3.core.result.RowView
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/pedagogical-document")
+class PedagogicalDocumentController(
+    private val accessControl: AccessControl,
+) {
+    @PostMapping
+    fun createPedagogicalDocument(
+        db: Database.Connection,
+        user: AuthenticatedUser,
+        @RequestBody body: PedagogicalDocumentPostBody
+    ): ResponseEntity<PedagogicalDocument> {
```

Comment:
ResponseEntity wrapping is completely unnecessary, so this would IMHO be a bit cleaner by just returning `PedagogicalDocument` directly